### ISSUE TITLE
[Okta] Integration can create error `field [okta.debug_context.debug_data.flattened.risk] does not contain value_split [=]`

### ISSUE THREAD

**Issue by BenB196** [elastic/integrations#4783]
Looking at some of the events that throw this error, it looks like Okta field `okta.debug_context.debug_data.flattened.risk` can be in the form of a weird array that the current Elastic Ingest Pipeline KV doesn't know how to handle.

https://github.com/elastic/integrations/blob/f9afcc0958180d8eb970a375d87de8d8fdf90864/packages/okta/data_stream/system/elasticsearch/ingest_pipeline/default.yml#L278-L283

Example of `okta.debug_context.debug_data.flattened.risk` that cause this error:

`"risk": "reasons=Anomalous Location, Anomalous Device, level=HIGH"`

**Comment by BenB196** [comment]
Hi @efd6 below is the `event.original`, I've replaced sensitive information with generic placeholder text.

```json
{
    "actor": {
        "alternateId": "Snipped_User@domain.com",
        "detailEntry": null,
        "displayName": "Last_name, First_Name",
        "id": "user_id",
        "type": "User"
    },
    "authenticationContext": {
        "authenticationProvider": null,
        "authenticationStep": 0,
        "credentialProvider": null,
        "credentialType": null,
        "externalSessionId": "102qmxOh1EdTHqn1_86CB9fzA",
        "interface": null,
        "issuer": null
    },
    "client": {
        "device": "unknown",
        "geographicalContext": {
            "city": "City",
            "country": "Country",
            "geolocation": {
                "lat": 00.00,
                "lon": 00.00
            },
            "postalCode": "00000",
            "state": "State"
        },
        "id": null,
        "ipAddress": "123.123.123.123",
        "userAgent": {
            "browser": "unknown",
            "os": "unknown",
            "rawUserAgent": "unknown"
        },
        "zone": "null"
    },
    "debugContext": {
        "debugData": {
            "authnRequestId": "Y5elHFMngoYoVKvakwnp2wAAAKo",
            "behaviors": "{New Geo-Location=NEGATIVE, New Device=POSITIVE, New IP=NEGATIVE, New State=NEGATIVE, New Country=NEGATIVE, Velocity=POSITIVE, New City=NEGATIVE}",
            "dtHash": "751b157a5a24ed83129433243e8d42307434b047120c32d7a7f5a5d2d91726fa",
            "requestId": "Y5elHFMngoYoVKvakwnp2wAAAKo",
            "requestUri": "/api/v1/authn",
            "risk": "{reasons=Anomalous Device, Anomalous Location, level=HIGH}",
            "threatSuspected": "false",
            "url": "/api/v1/authn?"
        }
    },
    "device": null,
    "displayMessage": "Verify user identity",
    "eventType": "user.authentication.verify",
    "legacyEventType": null,
    "outcome": {
        "reason": null,
        "result": "SUCCESS"
    },
    "published": "2022-12-12T22:03:08.791Z",
    "request": {
        "ipChain": [{
                "geographicalContext": {
                    "city": "City",
                    "country": "Country",
                    "geolocation": {
                        "lat": 00.00,
                        "lon": 00.00
                    },
                    "postalCode": "00000",
                    "state": "State"
                },
                "ip": "123.123.123.123",
                "source": null,
                "version": "V4"
            }
        ]
    },
    "securityContext": {
        "asNumber": 1828,
        "asOrg": "org",
        "domain": "domain.com",
        "isProxy": false,
        "isp": "isp"
    },
    "severity": "INFO",
    "target": null,
    "transaction": {
        "detail": {},
        "id": "Y5elHFMngoYoVKvakwnp2wAAAKo",
        "type": "WEB"
    },
    "uuid": "c32ae8ec-7a68-11ed-b8a7-9134a086ef85",
    "version": "0"
}

```

**Comment by efd6** [comment]
@BenB196 Thanks.
